launch/config.yml

```
@@ -0,0 +1,47 @@
+relaychain:
+  image: parity/polkadot:v0.9.24
+  chain: rococo-local
+  runtimeGenesisConfig:
+    configuration:
+      config:
+        validation_upgrade_cooldown: 10
+        validation_upgrade_delay: 10
+  env:
+    RUST_LOG: parachain::candidate-backing=trace,parachain::candidate-selection=trace,parachain::pvf=trace,parachain::collator-protocol=trace,parachain::provisioner=trace
+  flags:
+    - --rpc-methods=unsafe
+    - --wasm-execution=compiled
+    - --execution=wasm
+  nodes:
+    - name: alice
+    - name: bob
+
+parachains:
+- image: mangatasolutions/mangata-node:dev
```

```suggestion
- image: mangatasolutions/mangata-node:dev
  env:
    RUST_LOG: error
```
to disable logging at the time the chainspecs are generated